src/sql/platform/accounts/common/accountViewModel.ts

```diff
@@ -24,7 +25,11 @@ export class AccountViewModel {
 	private _updateAccountListEmitter: Emitter<UpdateAccountListEventParams>;
 	public get updateAccountListEvent(): Event<UpdateAccountListEventParams> { return this._updateAccountListEmitter.event; }
 
-	constructor(@IAccountManagementService private _accountManagementService: IAccountManagementService, @ILogService private _logService: ILogService) {
+	constructor(
+		@IAccountManagementService private _accountManagementService: IAccountManagementService,
+		@ILogService private _logService: ILogService,
+		@IAuthenticationService private readonly _authenticationService: IAuthenticationService
```

Comment:
nit: Use of readonly should be consistent within a file
```suggestion
		@IAccountManagementService private readonly _accountManagementService: IAccountManagementService,
		@ILogService private readonly _logService: ILogService,
		@IAuthenticationService private readonly _authenticationService: IAuthenticationService
```